server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java

```diff
@@ -342,6 +342,9 @@ public void childEvent(
       Thread.sleep(50);
     }
 
+    // wait historical data to be updated
+    Thread.sleep(druidCoordinatorConfig.getCoordinatorPeriod().getMillis());
```

Comment:
how long is the default here? we dont want unit tests that take minutes to complete